src/commands/synthetics/utils/internal.ts

```diff
@@ -87,3 +88,97 @@ export const toExecutionRule = (env: string | undefined): ExecutionRule | undefi
 
   return undefined
 }
+
+type validTestOverridesValues = 'boolean' | 'number' | 'string' | 'string[]' | 'ExecutionRule'
+type AccumulatorBaseConfigOverride = Omit<
+  BaseConfigOverride,
+  'basicAuth' | 'headers' | 'cookies' | 'deviceIds' | 'locations' | 'tunnel' | 'variables' | 'retry' // These options will be implemented later in separate PRs, see https://datadoghq.atlassian.net/browse/SYNTH-12971
+> & {
+  'retry.count'?: number
+  'retry.interval'?: number
+}
+type AccumulatorBaseConfigOverrideKeys = keyof AccumulatorBaseConfigOverride
+type OverridesValues = boolean | number | string | string[] | ExecutionRule
```

Comment:
I feel like there must be better names for these 4 but i can't personally come up with any so 🤷 :D
@Drarig29 is usually pretty good at this might be a good idea to get his opinion 🤔